content/docs/examples/advanced-gateways/egress-gateway/index.md

```diff
@@ -655,6 +655,18 @@ external service.
 
 1.  Follow the steps in the [Cleanup HTTPS gateway](#cleanup-https-gateway) section.
 
+## Enable Egress Gateway
```

Comment:
I don't think this belongs here. It's an awkward fit in the section flow.
How about adding something about how to enable egressgateway in the Install instructions and reference that from the before you begin section?

@vadimeisenbergibm WDYT?

Reply:
the question is why would someone wants to read enable egress gateway when they are just doing install...  I'm open to ideas on where it should belong to :)

Reply:
I know the install instructions need and are in the process of getting a lot of changes, but one thing I think is missing in the docs is what the various profiles (minimal, demo, standard) includes by default. For example the minimal install doc needs to say more than `This minimal install provides traffic management features of Istio`. What exactly is/isn't included. The "traffic management features" sort of implies the egressgateway would be included. I see that in the [helm option doc](https://preliminary.istio.io/docs/reference/config/installation-options/), it says egressgateway is enabled by default, so we probably need to clarify what "default" means.

Some of the install instructions are also missing a "Verify Installation" step that does the `kubectl get po` to confirm all the expected components are up and running. Anyway, I think our install instructions need a lot of work including somewhere reasonable to reference from examples that tell people about how to enable/disable the standard addons, like egressgateway.

Reply:
so the helm option doc will need to be refreshed... (cc @johnma14) 

egress gateway is off by default, unless you use demo profile. That is a change went in a week ago.

Reply:
I agree, if there is a page that talks about these are default components, and if you want to enable these components, here is what you do.  The current disabled (or to be disabled) components in v1.1 are istio-pilocy and istio-egressgateway and a few addons.

Reply:
@linsun The recent change to the install docs made several issues very clear. We definitely need to add the guidance that you and @frankbu suggest, but it is important that we do so in a consistent and systematic way to ensure we provide correct information without adding too much maintenance overhead. 
Feel free to check out some of the bugs I've already filed and comment.
https://github.com/istio/istio.io/issues/3431
https://github.com/istio/istio.io/issues/3432
https://github.com/istio/istio.io/issues/3433

@frankbu discussed it and he is adding an issue specifically for this concern.

Reply:
#3460

Reply:
@linsun @frankbu The issue is simple, you do not need to instal/update the whole Istio. You just deploy the egress gateway alone, by the following command:

```
$ helm template $HOME/istio-fetch/istio --name istio-egressgateway --namespace istio-system -x charts/gateways/templates/deployment.yaml -x charts/gateways/templates/service.yaml -x charts/gateways/templates/serviceaccount.yaml -x charts/gateways/templates/autoscale.yaml -x charts/gateways/templates/clusterrole.yaml -x charts/gateways/templates/clusterrolebindings.yaml --set global.istioNamespace=istio-system --set gateways.istio-ingressgateway.enabled=false --set gateways.istio-egressgateway.enabled=true | kubectl apply -f -
serviceaccount "istio-egressgateway-service-account" created
clusterrole "istio-egressgateway-istio-system" created
clusterrolebinding "istio-egressgateway-istio-system" created
service "istio-egressgateway" created
deployment "istio-egressgateway" created
horizontalpodautoscaler "istio-egressgateway" created
```

Reply:
I think we can do something better in the future, but maybe a good solution for now is to just add the command to the {{< warning >>}}, above.
```
{{< warning >}}
If the egress gateway is not enabled, run the following command to enable it:
{{< text bash >}}
$ helm template $HOME/istio-fetch/istio --name istio-egressgateway --namespace istio-system -x charts/gateways/templates/deployment.yaml -x charts/gateways/templates/service.yaml -x charts/gateways/templates/serviceaccount.yaml -x charts/gateways/templates/autoscale.yaml -x charts/gateways/templates/clusterrole.yaml -x charts/gateways/templates/clusterrolebindings.yaml --set global.istioNamespace=istio-system --set gateways.istio-ingressgateway.enabled=false --set gateways.istio-egressgateway.enabled=true | kubectl apply -f -
{{< /text >}}
{{< /warning >}}
```

Reply:
@frankbu I think that is probably good for now I will make the editing.

Reply:
validated the cmd runs good on my env. :)